lib/core/src/msParam.cpp

```diff
@@ -1451,60 +1451,61 @@ int
 addKeyValToMspStr( msParam_t * keyStr, msParam_t * valStr,
                    msParam_t * msKeyValStr ) {
 
-    if ( ( keyStr == NULL && valStr == NULL ) || msKeyValStr == NULL ) {
+    if ((keyStr == nullptr && valStr == nullptr) || msKeyValStr == nullptr) {
         return SYS_INTERNAL_NULL_INPUT_ERR;
     }
 
-    if ( msKeyValStr->type == NULL ) {
-        fillStrInMsParam( msKeyValStr, NULL );
+    if (msKeyValStr->type == nullptr) {
+        fillStrInMsParam(msKeyValStr, nullptr);
     }
 
-    char *keyPtr = parseMspForStr( keyStr );
-    int keyLen;
-    if ( keyPtr == NULL || strcmp( keyPtr, MS_NULL_STR ) == 0 ) {
-        keyLen = 0;
-    }
-    else {
-        keyLen = strlen( keyPtr );
-    }
+    auto get_length{[](const char* str) -> size_t {
+        if (str == nullptr || std::strcmp(str, MS_NULL_STR) == 0) {
+	    return 0;
+	}
+	return std::strlen(str);
+    }};
```

Comment:
Formatting.

Reply:
I don't see it, and I don't think 4.2 has clang format, so I can't cheat and use that. Are you saying the `if` should be split at the `||` or is it something else?

Reply:
Perhaps your editor is hiding the issue.

Try looking at those lines in your browser.

Reply:
The indentation is incorrect.

Reply:
i see tabs, and we're not doing tabs in irods/irods i believe...

Reply:
Correct.

Reply:
Should be good, wouldn't be surprised if more formatting errors showed up though. Haven't properly set up my new editor yet.